server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/deltas.ts

```diff
@@ -22,16 +22,22 @@
     tenantManager: ITenantManager,
     deltaService: IDeltaService,
     appTenants: IAlfredTenant[],
-    throttler: IThrottler): Router {
+    throttler: IThrottler,
```

Comment:
>throttler [](http://example.com/codeflow?start=4&length=9)

(nit) use tenantThrottler to be consistent?

Reply:
Done. :)